### PR TITLE
Fix Currency Config Loading

### PR DESCRIPTION
**Pull request by tn3rb**
This PR was cherry picked over from the Admin Refresh branch where it was added to fix failing unit tests.
Same tests are failing in dev branch.

### CODE REVIEW

File: core/EE_Config.core.php [hunk 1]
```diff
@@ -2141,39 +2142,82 @@ public function __construct($CNT_ISO = '')
...
-                $this->sign_b4 = $country->currency_sign_before(
-                );        // currency sign before or after: $TRUE  or  FALSE$
-                $this->dec_plc = $country->currency_decimal_places();    // decimal places: 2 = 0.00  3 = 0.000
-                $this->dec_mrk = $country->currency_decimal_mark(
-                );    // decimal mark: (comma) ',' = 0,01   or (decimal) '.' = 0.01
-                $this->thsnds = $country->currency_thousands_separator(
-                );    // thousands separator: (comma) ',' = 1,000   or (decimal) '.' = 1.000
-            }
-        }
+        $this->setCurrency($CNT_ISO);
```
Comment (knazart) [inline core/EE_Config.core.php:2145]: 👍

File: core/helpers/EEH_Money.helper.php [hunk 2]
```diff
@@ -159,79 +159,88 @@ public static function compare_floats($float1, $float2, $operator = '=')
...
      * @return string
      * @throws EE_Error
+     * @throws ReflectionException
      */
-    public static function get_format_for_jqplot($CNT_ISO = '')
+    public static function get_format_for_jqplot(?string $CNT_ISO = ''): string
     {
         // default format
-        $format = 'f';
-        $currency_config = $currency_config = EEH_Money::get_currency_config($CNT_ISO);
```
Comment (knazart) [inline core/helpers/EEH_Money.helper.php:171]: 🤔

**Review by knazart — APPROVED** [review]
Looks 👌 to me.

File: core/helpers/EEH_Money.helper.php (hunk 2 above)
Comment (tn3rb) [reply core/helpers/EEH_Money.helper.php:171]: just making sure the variable was set? 🤷🏻‍♂️

**Pull request merged**